website/docs/user_guide.md

```diff
@@ -567,3 +567,24 @@ needs extra work on the part of the plugin developer to make them useful in the
 GUI, some plugins require hand editing of configuration files).
 
 <Figure caption="Screenshot showing the plugin store inside the app" src="/img/plugin_store.png" />
+
+## Using the bookmark widget
+
+JBrowse Web and JBrowse Desktop by default come with a "bookmark widget"
+
+<Figure caption="Clicking and dragging on a region can be used to create a bookmark that can be returned to later on" src="/img/bookmark_widget.png"/>
+
+The bookmark stores a list of single regions (chromosome, start, and end
```

Comment:
Side note: is there a way to open the bookmark widget form the File dropdown? or can you only open it from the Header hamburger menu list? If there is another way to open it, we should document it?

Reply:
you can open it from the view menu (e.g. the hamburger menu on the view) also. i did not mention that in the docs here

Reply:
could update to include it if interested

Reply:
I think it would be a good thing to add to the docs. It would help users discover some of our hidden functionality or keep track of where to open these widgets if someone accidentally closes it.

Reply:
Added a modified figure that uses pink instead of red to highlight that you can open it also through the hamburger menu
![bookmark_widget (1)](https://user-images.githubusercontent.com/6511937/155245350-a8c41acd-ed9e-4654-a57c-65da49f5b995.png)